app/src/main/java/org/fossasia/susi/ai/data/device/DeviceServiceImpl.kt

```diff
@@ -83,8 +77,6 @@ class DeviceServiceImpl : DeviceService {
             override fun onFailure(call: Call<AddRoomResponse>, t: Throwable) {
                 if (t?.localizedMessage != null) {
```

Comment:
Why have you even used this statement. Timber.e shows the entire stack trace. You have not read about this 

Why making simple error logging so complicated?

Reply:
This method was implemented by @batbrain7. So, I followed the pattern that was used earlier.

Reply:
If I followed whatever was done before me, do you think I could have ever improved it or myself. If you want to be better, then you have to question existing stuff and find better solutions, and not just blindly copy something and follow the pattern

Reply:
I will surely remember this and try to improve more and more.